lib/triggers/antispamTrigger.js

```diff
@@ -0,0 +1,168 @@
+var util = require("util");
+var BaseTrigger = require("./baseTrigger.js").BaseTrigger;
+/*
+Punishes spammers. Assigns a penalty to every message in a groupchat, and when a certain score is reached, it penalizes the user.
+Options:
+ignores - array of strings([]) = users that won't be penalized (this trigger will effectively not see them). You probably want to put yourself here. (this is a built-in)
+logLevel - string(info) = what level should we log at? Choose a level from the top of chatBot.js
+admins - array of strings([]) = admins that the bot tattles to. You probably want to put yourself here. This function doesn't differentiate between users and groupchats, so you can have it report to a groupchat as well.
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/968187188/issues/source?bid=1980036&fileBranchId=2848073#l8)